app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -16,9 +14,10 @@ import {
 
 const NoteForm = ({ onClose, refetch, note, isEdit }) => {
   const [submitted, setSubmitted] = useState(false);
-
   const handleSubmit = async values => {
     try {
+      // eslint-disable-next-line no-console
+      console.log(values);
```

Comment:
This just wrong. You should not commit any file that has `console.log` statements. That's why we have added the eslint rules. If you are disabling those rules, then it defeats the whole purpose of those rules.

Reply:
I think i have rectified these statements in later PR.